src/ng/sniffer.js

```diff
@@ -61,7 +65,7 @@ function $SnifferProvider() {
       // so let's not use the history API also
       // We are purposefully using `!(android < 4)` to cover the case when `android` is undefined
       // jshint -W018
-      history: !!($window.history && $window.history.pushState && !(android < 4) && !boxee),
+      history: !!(hasHistoryPushState && !(android < 4) && !boxee),
```

Comment:
The thing is that we shouldn't even try to touch `history.pushState` in packaged apps (or there'll be errors).
That's wh ywe need the `!isChromePackagedApp` short-circuiting at the beginning of `hasHistoryPushState`.